### PR TITLE
Feature/c100 rebuild prlc100 175

### PR DESCRIPTION
**Pull request by monalisasarkar1**
### JIRA link (if applicable) ###
 https://tools.hmcts.net/jira/browse/PRLC100-175

### Change description ###
ask the applicant question about if they have a document signed by a mediator, if they have indicated that they have attended MIAM previously 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/miam/mediator-document/content.test.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    const form = generatedContent.form as FormContent;
+    const fields = form.fields as FormFields;
+    const applyingWithField = fields.haveDocSigned as FormOptions;
+    expect(applyingWithField.type).toBe('radios');
+    expect(applyingWithField.classes).toBe('govuk-radios');
+    expect((applyingWithField.values[0].label as LanguageLookup)(generatedContent)).toBe(en.one);
+    expect((applyingWithField.values[1].label as LanguageLookup)(generatedContent)).toBe(en.two);
+  });
+
+  test('should contain saveAndComeLater button', () => {
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/mediator-document/content.test.ts:47]: @monalisasarkar1 Can you please add test case for Continue button ?.

File: src/main/steps/c100-rebuild/miam/mediator-document/template.njk [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% extends "common/template.njk" %}
+
+
+{% block page_content %}
+<span class="govuk-caption-xl">{{caption}}</span>
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/mediator-document/template.njk:5]: @monalisasarkar1 Have you defined caption in content.ts ?
If not then please remove this unused line.

File: src/main/steps/c100-rebuild/miam/mediator-document/content.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  two: 'No',
+  errors: {
+    haveDocSigned: {
+      required: 'Select yes if you have a document signed by the mediator',
+    },
+  },
+});
+
+const cy = () => ({
+  headingTitle: 'Do you have a document signed by the mediator? - welsh  ',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/mediator-document/content.ts:20]: @monalisasarkar1 Can we rename 'headingTitle' to 'title' ?

File: src/main/steps/c100-rebuild/miam/mediator-document/content.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    haveDocSigned: {
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/mediator-document/content.ts:39]: Can we please prefix c1A_ for this field ? (c1A_haveDocSigned)

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/miam/mediator-document/content.test.ts (hunk 1 above)
Comment (monalisasarkar1) [reply src/main/steps/c100-rebuild/miam/mediator-document/content.test.ts:47]: done

File: src/main/steps/c100-rebuild/miam/mediator-document/template.njk (hunk 2 above)
Comment (monalisasarkar1) [reply src/main/steps/c100-rebuild/miam/mediator-document/template.njk:5]: done

File: src/main/steps/c100-rebuild/miam/mediator-document/content.ts (hunk 3 above)
Comment (monalisasarkar1) [reply src/main/steps/c100-rebuild/miam/mediator-document/content.ts:20]: done

File: src/main/steps/c100-rebuild/miam/mediator-document/content.ts (hunk 4 above)
Comment (monalisasarkar1) [reply src/main/steps/c100-rebuild/miam/mediator-document/content.ts:39]: done

**Review by channa-patil — APPROVED** [review]

**Review by channa-patil — APPROVED** [review]

**Pull request merged**